cmake/recipes/sympiler.cmake

```diff
@@ -24,17 +24,21 @@ else()
 endif()
 
 if(SYMPILER_BLAS_BACKEND STREQUAL "MKL")
+    message("INCLUDING MKL")
     include(mkl)
+    if(NOT TARGET MKL::MKL)
+        add_library(MKL::MKL ALIAS mkl::mkl)
+    endif()
 endif()
 
 include(CPM)
 CPMAddPackage(
     NAME sympiler
     GITHUB_REPOSITORY ryansynk/sympiler
     GIT_TAG 51bffd948f902b4606b8a8173a933ad9b54e29bf
```

Comment:
Don't forget to update your fork with our changes to sympiler!

Reply:
Just got that updated!